packages/nextlove/src/generate-openapi/index.ts

```diff
@@ -225,13 +241,16 @@ export async function generateOpenAPI(opts: GenerateOpenAPIOpts) {
       }
     }
 
-    route.operationId = transformPathToOperationId(routePath)
-
     // Some routes accept multiple methods
     builder.addPath(routePath, {
       ...methods
         .map((method) => ({
-          [method.toLowerCase()]: route,
+          [method.toLowerCase()]: {
+            ...route,
+            operationId: `${transformPathToOperationId(routePath)}${
+              methods.length > 1 ? pascalCase(method) : ""
```

Comment:
i think we might want to always include the method- it would be weird on Seam Connect for them to be labeled differently if they happen to only have one method